app/views/docs/getting-started-for-web.phtml

```diff
@@ -62,20 +62,11 @@ $demos = $platform['demos'] ?? [];
 <p>Initialize your SDK code with your project ID which can be found in your project settings page.</p>
 
 <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
-    <pre class="line-numbers"><code class="prism language-javascript" data-prism>const client = new Client();
-
-// Init your Web SDK
-client
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>const client = new Client()
     .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
-    .setProject('5df5acd0d48c2') // Your project ID
-;</code></pre>
+    .setProject('5df5acd0d48c2');               // Your project ID
```

Comment:
This applies to all code blocks:

when `</code></pre>` is put on the next line, it actually adds an extra blank line after the code examples. We should have `</code></pre>` after the last character in the code example, not after the last line :)